src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -150,4 +152,32 @@ public void updateBadId() throws Exception {
                 .andExpect(status().is4xxClientError());
 
     }
+
+    @Test public void findNotFound() throws Exception {
+        mvc.perform(get("/api/books/search?isbn=sarasa"))
+                .andExpect(status().is4xxClientError());
+    }
+
+    @Test public void findByIsbnFoundOpenLibrary() throws Exception {
+        String isbnInOpenLibrary = "0385472579";
+        mvc.perform(get("/api/books/search?isbn=" + isbnInOpenLibrary))
+                .andExpect(status().isCreated());
```

Comment:
You could analyze if the JSON returned by the search is correct. At least checking the ISBN field that's being returned.